packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/DropdownDemo/DropdownDemo.tsx

```diff
@@ -46,7 +46,8 @@ export class DropdownDemo extends React.Component<{}, DropdownState> {
         isOpen
       });
     };
-    this.onSelect = event => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    this.onSelect = _event => {
```

Comment:
or change to no params `() => {}`

Reply:
Suggest we keep the params to show what is expected here, considering this is a demo